src/MergeContext.js

```diff
@@ -337,19 +346,47 @@ class MergeContext {
         const userRequested = usersVoted.find(el => el.state === 'changes_requested');
         if (userRequested !== undefined) {
             this._log("changes requested by " + userRequested.reviewer);
-            return null;
+            this._approvalResult.params = {desc: "blocked (see change requests)", state: "error"};
+            return;
         }
         const usersApproved = usersVoted.filter(u => u.state !== 'changes_requested');
         this._log("approved by " + usersApproved.length + " core developer(s)");
 
         if (usersApproved.length < Config.necessaryApprovals()) {
             this._log("not approved by necessary " + Config.necessaryApprovals() + " votes");
-            return null;
+            this._approvalResult.params = {desc: "waiting for more votes", state: "pending"};
+            return;
+        }
+        if (usersApproved.length >= Config.sufficientApprovals()) {
+            this._approvalResult.params = {desc: "approved", state: "success"};
```

Comment:
It is strange that approved PRs do not have a `_log()` call while other `if`s have it. However, it may be better to simply log the `_approvalResult` upon this method return (in all cases).